Formula/dnscontrol.rb

```diff
@@ -6,9 +6,12 @@ class Dnscontrol < Formula
   license "MIT"
   version_scheme 1
 
+  # Upstream appears to use GitHub releases to indicate that a version is
+  # released (and they also recall some of the unreleased tags in the past), so it's
+  # necessary to check release versions instead of tags.
```

Comment:
```suggestion
  # released and they sometimes re-tag versions before that point, so it's
  # necessary to check release versions instead of tags.
```

I was initially unsure what "recall" entailed here but it sounds like it's just re-tagging a version. Explaining it that way may be clearer to other maintainers, so I've tweaked the language here.

Reply:
yeah, from the tag reuse perspective, it could be considered a re-tagging, but the upstream kind of abused it by `recall and re-tag` IMO 😅